pages/docs/setup/connect.mdx

```diff
@@ -0,0 +1,175 @@
+- **Elastic horizontal scaling** - Easily add more capacity by running additional workers.
+- **Ideal for container runtimes** -  Deploy on Kubernetes or ECS without the need of a load balancer for inbound traffic
+- **Simpler long running steps** - Step execution is not bound by platform http timeouts.
+
+## Getting started
+
+Using `connect` with your app is simple. Using each SDK's "connect" method only requires a list of functions that are available to be executed. (NOTE - Python is not yet supported)
+
+Here is a one-file example of a fully-functioning app that connects to Inngest.
+
+<CodeGroup>
+```ts
+import { Inngest } from 'inngest'
+
+const inngest = new Inngest({
+  id: 'my-app'
+});
+
+const handleSignupFunction = inngest.createFunction(
+  { id: 'handle-signup' },
+  { event: 'user.created'}
+  async ({ event, step }) => {
+    console.log('Function called', event);
+  }
+);
+
+inngest.connect({
+  functions: [handleSignupFunction],
+}).then((connection) => {
+  console.log('Worker: connected', connection);
+});
+```
+```go
+type UserCreatedEvent struct {
+	Name string
+	Data struct {
+		UserID string `json:"user_id"`
+	}
+}
+
+func main() {
+	ctx := context.Background()
+
+	app := inngestgo.NewHandler("my-app", inngestgo.HandlerOpts{
+		Logger:  logger.StdlibLogger(ctx),
+	})
+
+	f := inngestgo.CreateFunction(
+		inngestgo.FunctionOpts{ID: "handle-signup", Name: "Handle signup"},
+		inngestgo.EventTrigger("user.created", nil),
+		func(ctx context.Context, input inngestgo.Input[UserCreatedEvent]) (any, error) {
+      fmt.Println("Function called")
+			return map[string]any{"success": true}, nil
+		},
+	)
+
+	app.Register(f)
+
+	fmt.Println("Worker: connecting")
+
+	err := app.Connect(ctx)
+	if err != nil {
+		fmt.Printf("ERROR: %#v\n", err)
+		os.Exit(1)
+	}
+}
+```
+</CodeGroup>
+
+## How does it work?
+
+The `connect` API establishes a persistent WebSocket connection to Inngest. Each connection can handle executing multiple functions and steps concurrently. Each app can create multiple connections to Inngest enabling horizontal scaling. Additionally, connect has the following features:
+
+- **Automatic re-connections** - The connection will automatically reconnect if it is closed.
+- **Graceful shutdown** - The connection will gracefully shutdown when the app receives a signal to terminate (`SIGTERM`). New steps will not be accepted after the connection is closed, and existing steps will be allowed to complete.
+- **Worker-level maximum concurrency** - Each worker can configure the maximum number of concurrent steps it can handle. This allows Inngest to distribute load across multiple workers and not overload a single worker.
+
+## Deploying to production
+
+<Steps>
+  <Step title="Set signing and event keys">
+    To enable your application to securely connect to Inngest, you must set the `INNGEST_SIGNING_KEY` and `INNGEST_EVENT_KEY` environment variables.
```

Comment:
The signing key alone is sufficient for using connect, though I understand if you want to use the event key for sending events